File: backend/src/main/java/com/mulesoft/arc/arcdatastore/backend/models/ArcAnalyticsMonthlySessions.java
package com.mulesoft.arc.arcdatastore.backend.models;

/**
 * A class representing Daily Sessions type.
 */

public class ArcAnalyticsMonthlySessions extends ArcAnalyticsRangeResult {

    public ArcAnalyticsMonthlySessions(String day) {
        super("ArcAnalytics#MonthlySessions", day);
    }
}

File: models/arc-analytics-monthly-sessions.js
'use strict';

const {ArcAnalyticsRangeResult} = require('./arc-analytics-range-result');

/**
 * A class representing monthly Sessions type.
 */
class ArcAnalyticsMonthlySessions extends ArcAnalyticsRangeResult {
  /**
   * @param {String} startDay The start day of the date range.
   * @param {String} endDay The last day of the date range.
   * @param {Number} result The result of computation.
   */
  constructor(startDay, endDay, result) {
    super('ArcAnalytics#MonthlySessions', startDay, endDay, result);
  }
}

module.exports.ArcAnalyticsMonthlySessions = ArcAnalyticsMonthlySessions;
